ingest/expression_files/dense_ingestor.py

```diff
@@ -28,6 +28,8 @@
 class DenseIngestor(GeneExpression, IngestFiles):
     # ToDo SCP-2635
     ALLOWED_FILE_TYPES = ["text/csv", "text/plain", "text/tab-separated-values"]
```

Comment:
Batch sizes values aren't particular to dense matrices but to all expression data. I think these values should move to the GeneExpression class.

Reply:
agree